components/script/dom/performance.rs

```diff
@@ -111,7 +110,6 @@ struct PerformanceObserver {
 #[dom_struct]
 pub struct Performance {
     reflector_: Reflector,
```

Comment:
This needs to be `eventtarget: EventTarget` instead now ([docs](https://doc.servo.org/script/dom/index.html#inheritance)).